geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/AcceptorImpl.java

```diff
@@ -1522,6 +1487,47 @@ public void run2() {
     }
   }
 
+  private CommunicationMode getCommunicationMode_notSelector(Socket socket) throws IOException {
```

Comment:
Generally underscores are not part of method names.
Could we possibly rename the method name to `getCommunicationModeForNonSelectorSocket`